src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -248,7 +249,7 @@ internal bool TryWrite(Utf8JsonWriter writer, T value, JsonSerializerOptions opt
 
                 int originalPropertyDepth = writer.CurrentDepth;
 
-                Write(writer, value, options);
+                Write(writer, value!, options);
```

Comment:
Can you add a link to the issue in all these places as a reminder?
```C#
// TODO: https://github.com/dotnet/runtime/issues/32523
```

Reply:
Sure, I'll add that.